orca-core/src/test/groovy/com/netflix/spinnaker/orca/pipeline/model/ExecutionSpec.groovy

```diff
@@ -17,7 +17,7 @@
 package com.netflix.spinnaker.orca.pipeline.model
 
 import com.netflix.spinnaker.security.AuthenticatedRequest
-import org.apache.log4j.MDC
+import org.slf4j.MDC
```

Comment:
Production code was already using `org.slf4j.MDC` but the tests weren't and the Apache one is no longer on the classpath.